internal/common/common.go

```diff
@@ -303,3 +303,12 @@ func CreateTestSignedTag(t *testing.T, repo *git.Repository, tagName string, tar
 
 	return tagHash
 }
+
+// ReverseSlice reverses a slice of any type using generics.
+func ReverseSlice[T any](slice []T) []T {
```

Comment:
can we just use https://pkg.go.dev/slices#Reverse?

Reply:
Fixed